### PR TITLE
the gateway room now has glass shutters, gateway lights up only when theres an active location

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

the gateway rooms on all stations now use glass instead of regular shutters
tram now has gateway shutters
the gateway's lights only light up when it finds a location thats available

## Why It's Good For The Game

lets people know if a gateway is active, useful since were reenabling those.

## Changelog

:cl:
expansion: gateway now lights up when theres an active location. gateway rooms use glass shutters
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
why's the diffbot for deltastation so massive?

**Comment by Snowyypanda** [comment]
nice, more gateway updates, more please!

**Comment by Fikou** [comment]
> why's the diffbot for deltastation so massive?

image

File: code/modules/awaymissions/gateway.dm [hunk 1]
```diff
@@ -171,6 +171,8 @@ GLOBAL_LIST_EMPTY(gateway_destinations)
 	var/obj/effect/gateway_portal_bumper/portal
 	/// Visual object for handling the viscontents
 	var/obj/effect/gateway_portal_effect/portal_visuals
+	/// Overlay of the lights. They light up fully when it charges fully.
+	var/mutable_appearance/light_overlay
```
Comment (Mothblocks) [inline code/modules/awaymissions/gateway.dm:175]: Shouldn't you QDEL_NULL this on Destroy?

Comment (Fikou) [reply code/modules/awaymissions/gateway.dm:175]: no need

**Pull request merged**